Sofa/Component/SolidMechanics/Spring/src/sofa/component/solidmechanics/spring/RestShapeSpringsForceField.h

```diff
@@ -88,6 +90,8 @@ class RestShapeSpringsForceField : public core::behavior::ForceField<DataTypes>
 
     /// Add the forces.
     void addForce(const core::MechanicalParams* mparams, DataVecDeriv& f, const DataVecCoord& x, const DataVecDeriv& v) override;
+    /// Link to be set to the topology container in the component graph.
+    SingleLink<core::behavior::ForceField<DataTypes>, sofa::core::topology::BaseMeshTopology, BaseLink::FLAG_STOREPATH | BaseLink::FLAG_STRONGLINK> l_topology;
```

Comment:
I am not quite sure where this attribute is actually initialized.
Can we do that for the topology of the external Mechanical State as well to link to d_external_points?

Reply:
This is initialized at the same time as others Data, should be at construction time? after the parse but before the init ?

Reply:
But where does that happen for the RestShapeSpringsForceField? I cannot find it anywhere.